src/service/resource.js

```diff
@@ -493,12 +493,7 @@ export class Resource {
    * @return {!../layout-rect.LayoutRectDef}
    */
   getLayoutBox() {
-    if (!this.isFixed_) {
-      return this.layoutBox_;
-    }
-    const viewport = this.resources_.getViewport();
-    return moveLayoutRect(this.layoutBox_, viewport.getScrollLeft(),
-        viewport.getScrollTop());
+    return this.layoutBox_;
```

Comment:
I'm fairly certain this will break layout position reporting for InOb, possibly others.